### PR TITLE
Bug 1768507 Add monitoring table for live table duplicates

### PR DESCRIPTION
**Pull request by jklukas**
See https://bugzilla.mozilla.org/show_bug.cgi?id=1768507

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title)
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/monitoring_derived/duplicate_rate_v1/query.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+"""Generates counts of duplicate document_id values per table per day."""
```
Comment (jklukas) [inline sql/moz-fx-data-shared-prod/monitoring_derived/duplicate_rate_v1/query.py:3]: This python script is copied from the structured_missing_columns_v1 table and only lightly modified to suit this case.

File: sql/moz-fx-data-shared-prod/monitoring_derived/duplicate_rate_v1/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+friendly_name: Duplicate Rate
```
Comment (scholtzan) [inline sql/moz-fx-data-shared-prod/monitoring_derived/duplicate_rate_v1/metadata.yaml:2]: Renaming to something like "Live Duplicate Rate" might maybe make it clearer that this data is only for live tables?

**Review by scholtzan — APPROVED** [review]

File: sql/moz-fx-data-shared-prod/monitoring_derived/duplicate_rate_v1/metadata.yaml (hunk 2 above)
Comment (scholtzan) [reply sql/moz-fx-data-shared-prod/monitoring_derived/duplicate_rate_v1/metadata.yaml]: the results table name should be like `live_duplicate_rate_v1` too?

**Comment by jklukas** [comment]
I did a more comprehensive look through existing monitoring queries, and I think I prefer https://github.com/mozilla/bigquery-etl/pull/2958 instead of adding a new table here.

**Pull request closed without merging**